### PR TITLE
Give up maintenance of Jitsi

### PR DESCRIPTION
**Pull request by yalh76**

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -1292,7 +1292,7 @@
         "category": "communication",
         "level": 0,
         "revision": "HEAD",
-        "state": "working",
+        "state": "notworking",
```
Comment (kay0u) [inline apps.json:1295]: You really want to give up the maintenance ? If yes:

```suggestion
        "state": "notworking",
        "maintained": false,
```

**Comment by frju365** [comment]
Just a question : Is it difficult to maintain the package ? (it's just to know, because it's possible I maintain the package in future).

**Comment by yalh76** [comment]
> 
> 
> Just a question : Is it difficult to maintain the package ? (it's just to know, because it's possible I maintain the package in future).

The point is: even if that package was level 7, there is application issue: https://github.com/YunoHost-Apps/jitsi_ynh/issues/10, something around igasi and metronome Bosh

**Comment by Djip007** [comment]
Maybe with ynh-4 based en debian Buster that have package suport from debian il may be more simple...
https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-quickstart look more simple with it...

:crossed_fingers:

**Comment by yalh76** [comment]
> 
> 
> Maybe with ynh-4 based en debian Buster that have package suport from debian il may be more simple...
> https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-quickstart look more simple with it...
> 
> 🤞

No there won't be any difference, there was also package for Stretch....

**Comment by Djip007** [comment]
> This document describes the steps for a quick Jitsi-Meet installation on a Debian-based GNU/Linux system. The following distributions are supported out-of-the-box:
>
>   Debian 10 (Buster) or later
>   Ubuntu 18.04 (Bionic Beaver) or later

the quick start report only Buster++ for jitsi repo
where do you find Stretch package?

**Comment by yalh76** [comment]
https://jitsi.org/downloads/ubuntu-debian-installations-instructions/

**Comment by Djip007** [comment]
> This document describes the steps for a quick Jitsi-Meet installation on a Debian-based GNU/Linux system. The following distributions are supported out-of-the-box:
>
>   Debian 10 (Buster) or later
>   Ubuntu 18.04 (Bionic Beaver) or later

the quick start report only Buster++ for jitsi repo
where do you find Stretch package?

**Pull request merged**